codegenerator/cli/templates/dynamic/codegen/src/IO.res.hbs

```diff
@@ -232,17 +233,17 @@ module RollBack = {
       {{#each entities as | entity |}}
       | {previousEntity: Some({entity: {{entity.name.capitalized}}(entity), eventIdentifier}), entityId} =>
         inMemStore.{{entity.name.uncapitalized}}->InMemoryTable.Entity.set(
-          Set(entity)->Types.mkEntityUpdate(~eventIdentifier, ~entityId, ~shouldSaveHistory=false),
-          ~shouldRollbackOnReorg,
+          Set(entity)->Types.mkEntityUpdate(~eventIdentifier, ~entityId, ~shouldSaveHistory),
+          ~shouldSaveHistory,
```

Comment:
I realise that there is logic duplication here for "shouldSaveHistory" and I want to refactor that code.